samples/nrf5340/netboot/src/main.c

```diff
@@ -41,6 +44,20 @@ int main(void)
 	bool valid = false;
 	uint8_t status = pcd_fw_copy_status_get();
 
+#ifdef CONFIG_PCD_LOCK_NETCORE_DEBUG
```

Comment:
Can you use `IS_ENABLED` here instead?

Reply:
I think that `IS_ENABLED` would have worked great with the original, but it does not really work too well with swich-case.